docs/ReleaseNotes.md

```diff
@@ -3,6 +3,70 @@ id: ReleaseNotes
 title: Release Notes
 ---
 
+## 2020.10.0 - 10/27/2020
+
+### New Features
+
+#### Enklu Embedded
+
+Does your current workflow make you bounce between Unity and the web editor? Have you dreamt of a world where editing an asset in Unity would also edit that asset in the web editor?
+
+Introducing Enklu Embedded in 2020.10! Enklu Embedded allows for users to embed Enklu Cloud content into their Unity scenes. It also enables Unity developers the ability to sync 3D assets from Unity back to Enklu Cloud.
+
+With Enklu Embedded, Unity developers can enjoy the freedom of Unity with the rapid iteration and real-time synchronization of Enklu Cloud (talk about having your cake and eating it too!).
+
+For more information on Enklu Embedded, check out our official guide [here](Embedded/EmbeddedSetup).
+
+
+![Enklu Embedded](/img/product/embedded/VolumeUpdates.gif)
+
+
+
+#### Web Play Mode
+
+New to Enklu Cloud is Web Play Mode! Web Play Mode allows you to test your experience directly from the web editor, without needing to don a headset. You can switch to Web Play Mode via the Play button at the top of the canvas.
```

Comment:
"without needing to don a headset" I kind of don't like this phrase mostly because I didn't know what it meant, had to look it up, and it wasn't in the first search result 😅  Maybe instead it can say "without needing to use a headset"?